packages/unenv-preset/src/runtime/polyfill/performance.ts

```diff
@@ -1,30 +1,27 @@
 import {
 	performance,
-	// Note: `Performance` is only a type in `node:perf_hooks`
-	//       But it is a class in the polyfill, see `runtime/node/perf_hooks.ts`
 	Performance,
 	PerformanceEntry,
 	PerformanceMark,
 	PerformanceMeasure,
 	PerformanceObserver,
 	PerformanceObserverEntryList,
 	PerformanceResourceTiming,
-} from "node:perf_hooks";
+} from "unenv/node/perf_hooks";
```

Comment:
I'm going to check again to see if this change is still needed...

Reply:
At least it is not needed any more on the wrangler side.

Reply:
Confirmed that this is no longer needed.